main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"crypto/rsa"
+	"github.com/joho/godotenv"
+	"github.com/ufcg-lsd/arrebol-pb-worker/utils"
+	"github.com/ufcg-lsd/arrebol-pb-worker/worker"
+	"log"
+	"os"
+)
+
+const (
+	ConfFilePathKey = "CONF_FILE_PATH"
+	ServerEndpointKey = "SERVER_ENDPOINT"
+)
+
+func generateKeys(workerId string) {
+	log.Println("Starting to gen rsa key pair with workerid: " + workerId)
+	utils.GenAccessKeys(workerId)
+}
+
+//This functions sends the worker's public key to the server
+func sendKey(serverEndPoint string, workerId string) {
+	log.Println("Sending pub key to the server. ServerEndpoint: " + serverEndPoint)
+	url := serverEndPoint + "/workers/publicKey"
+	requestBody := &map[string]*rsa.PublicKey{"key": utils.GetPublicKey(workerId)}
+	httpResp := utils.Post(requestBody, url)
+
```

Comment:
what is best? to handle the error here or at one level above? @paulofelipefeitosa

Reply:
Apparently `utils.Post` panics on an error, this is the expected behavior? Should the worker stop its execution when something is wrong?

Reply:
I think utils.Post shouldn't handle the error, I've taken another approach in the getTask pr, where I handle it one level above, I'll create an issue to extract the error handling from the Post, so we avoid some conflicts and keep the pr size (that is not the best one, btw). Ok guys?

Reply:
ok